fr/integrations/plugin-packs/procedures/applications-monitoring-centreon-database.md

```diff
@@ -1,52 +1,239 @@
+| memory.buffer.bytes     | Bytes |
+| memory.cached.bytes     | Bytes |
+| memory.shared.bytes     | Bytes |
+
+<!--Swap-->
+
+| Métrique                    | Unité   |
+| :-------------------------- | :------ |
+| swap.usage.bytes            | Bytes   |
+| swap.free.bytes             | Bytes   |
+| swap.usage.percentage       | %       |
+
+<!--Load-->
+
+| Métrique                    | Unité                          |
+| :-------------------------- | :----------------------------- |
+| load1                       | System load 1 minute-sample    |
+| load5                       | System load 5 minutes-sample   |
+| load15                      | System load 15 minutes-sample  |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Prérequis
+
+### Création d'un utilisateur de base de données 
+
+Pour permettre la récolte d'informations liées à la base de données, la création d'un utilisateur avec des droits spécifiques est nécessaire :
+
+``` mysql
+CREATE USER 'monitor_user'@'IP_POLLER' IDENTIFIED BY 'a_very_secure_passwd';
+GRANT SELECT ON *.* to 'monitor_user'@'IP_POLLER';
+```
+
+### SNMP
+
+SNMP doit être configuré sur le serveur central. Vous pouvez vous aider de cette [documentation](../operatingsystems-linux-snmp.html#prerequisites) pour mettre en place rapidement une simple configuration SNMP. 
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online License-->
+
+1. Installez les plugins centreon suivant sur le collecteur qui supervisera la base de donnée :
+
+```bash
+yum install centreon-plugin-Applications-Monitoring-Centreon-Database centreon-plugin-Operatingsystems-Linux-Snmp
+
+```
+
+2. Sur l'interface Web de Centreon, installez le Plugin Pack **Centreon Databse** depuis la page **Configuration > Packs de plugins**.
+
+<!--Offline License-->
+
+1. Installez les plugins centreon suivant sur le collecteur qui supervisera la base de donnée :
+
+
+```bash
+yum install centreon-plugin-Applications-Monitoring-Centreon-Database centreon-plugin-Operatingsystems-Linux-Snmp
+```
+
+2. Sur le serveur Central Centreon, installez le RPM du Pack **Centreon Database** :
+
+
+ ```bash
+yum install centreon-pack-applications-monitoring-centreon-database
+```
+
+3. Sur l'interface Web de Centreon, installez le Plugin Pack **Centreon Databse** depuis la page **Configuration > Packs de plugins**.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Hôte
+
+* Ajoutez un Hôte à Centreon depuis la page **Configuration > Hôtes**.
+* Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre serveur *Centreon Database*.
+* Appliquez le Modèle d'Hôte **App-Monitoring-Centreon-Database-custom**.
+* Une fois le modèle appliqué, les Macros ci-dessous indiquées comme requises (*Obligatoire*) doivent être renseignées.
+
+| Mandatory  | Macro            |
+| :--------- | :--------------  |
+| X          | MYSQLPORT        |
+| X          | MYSQLUSERNAME    |
+| X          | MYSQLPASSWORD    |
+|            | SNMPEXTRAOPTIONS |

```

Comment:
Maybe (for EN too) it could be interesting to precise that the MYSQLUSERNAME and the MYSQLPASSWORD should correspond to the command "GRANT USER bla bla ?"

Reply:
I thought it was obvious, i added it 😃